server/website/website/tasks/periodic_tasks.py

```diff
@@ -18,20 +18,27 @@
                                     DummyEncoder,
                                     consolidate_columnlabels)
 from website.models import PipelineData, PipelineRun, Result, Workload
-from website.types import PipelineTaskType
+from website.types import PipelineTaskType, WorkloadStatusType
 from website.utils import DataUtil, JSONUtil
 
 # Log debug messages
 LOG = get_task_logger(__name__)
```

Comment:
I would change all (or most) of the log messages in this file from "info" to "debug" level (keep any messages you think should be "info" level)